packages/iov-crypto/src/bip39.ts

```diff
@@ -42,7 +46,9 @@ export class EnglishMnemonic {
 
 export class Bip39 {
   public static encode(entropy: Uint8Array): EnglishMnemonic {
-    if (entropy.length !== 16 && entropy.length !== 24 && entropy.length !== 32) {
+    const allowedEntropyLengths: ReadonlyArray<number> = [16, 20, 24, 28, 32]; // 20,28
```

Comment:
Can we make `allowedEntropyLengths` and `allowedWordsLengths` consistent? Either as a static variable or locally. Hard to say what is better. It was loclly before and is not reused at the moment. But I'm open here.